src/main.rs

```diff
@@ -163,18 +163,22 @@ Faster Nmap scanning with Rust."#;
     println!("{}", info.gradient(Color::Yellow).bold());
     funny_opening!();
 
-    let config_path = match dirs::config_dir() {
-        Some(mut path) => {
-            path.push("rustscan");
-            path.push("config.toml");
-            path
-        }
-        None => panic!("Couldn't find config dir."),
+    let mut config_dir = match dirs::config_dir() {
+        Some(dir) => dir,
+        None => panic!("Could not infer config file path."),
+    };
+    config_dir.push("rustscan");
+    config_dir.push("config.toml");
+
+    let mut home_dir = match dirs::home_dir() {
+        Some(dir) => dir,
+        None => panic!("Could not infer config file path."),
     };
+    home_dir.push(".rustscan.toml");
 
     detail!(format!(
```

Comment:
This is a bug I made, pretty pleasee can you fix it :pleading_face: 
This doesn't work with grep mode or accessible mode :(
Can you uhhhh get the config args (or config file struct) and fix my bug? 
See this code for the syntax.
https://github.com/RustScan/RustScan/blob/fa072bac1235fbbfe615893f56f992c4ae7b0cb0/src/tui.rs
detail!("message", greppable, accessible);

Reply:
oh i clicked the wrong thing. I meant the `detail!` you can see, but I clicked the `home_dir.push` :laughing:

Reply:
PS: if you do this we can officially say we are screen reader friendly which is something not even Nmap can say! :smile:

Reply:
Changed the `print_opening()` function to accept the opts struct, which now contains both the greppable and accessible options, so we can match on those with the `detail!()` macro. Now works fine in both cases.